app.js

```diff
@@ -2,12 +2,11 @@ const express = require("express");
 const app = express();
 const { getApi } = require("./controller/get-api.controller");
 const { getTopics } = require("./controller/topics.controller");
+const { handleCustomErr, handlePsqlErr } = require("./error-handling");
 const {
-  handleCustomErr,
-  handleServerErr,
-  handlePsqlErr,
-} = require("./error-handling");
-const { getArticleById } = require("./controller/articles.controller");
+  getArticleById,
+  getArticles,
+} = require("./controller/articles.controller");
 app.use(express.json());
```

Comment:
Unless your ticket is for a post or patch endpoint, please do not include `app.use(express.json())`. Only write the tests for the task and the code required to pass those tests.